### ISSUE TITLE
hello sir i am facing so many issue while scraping the dmart data can you please help me plz its my humble request to you can you tell me step by step what to do 

### ISSUE THREAD

**Issue by arpitdayma** [hemantam1/Infoware#1]

**Comment by AryanRajput78** [comment]
Please share your errors or Screenshot of them, so we can figure out what is going wrong.

**Comment by arpitdayma** [comment]
sir i am new in web scraping  and i want to scrap d-mart data from its website but i try your webscraper but its not works i dont know how to use it i just open your dmart_scraper.py in pycharm and its show some error  Sir can you make one small video how to use it it will really help me a lot . and i am a app devloper if you need any help i will do it. i am learning python only 

>
<img width="960" alt="Screenshot 2022-02-28 094417" src="https://user-images.githubusercontent.com/80608220/155923035-5261a969-ec2c-4ab1-a24b-286fc2096290.png">
<img width="960" alt="Screenshot 2022-02-28 094348" src="https://user-images.githubusercontent.com/80608220/155923039-357400fc-0beb-435c-8778-0c640b57f5f0.png">

**Comment by AryanRajput78** [comment]
This looks like a simple fix. Check the path to the chrome driver as it is known to cause such errors if not set up properly. Also, one more thing is to check the output file path so that when you run the script the data get stored at the location that you want.

![doubt](https://user-images.githubusercontent.com/46697997/155925422-bb2c31eb-9c29-4e80-93be-d820215bfe01.png)
Insert your path for the chrome driver here, and run the code.

**Comment by arpitdayma** [comment]
sir these scraper scrap data of one single product or  it can scrap of one full category?

**Comment by AryanRajput78** [comment]
As I have mentioned there are two different files in this repository. The Combos/offers and product scraping. Currently, both of them work separately. Specific category or page-wise scraping is not an option yet.

**Comment by arpitdayma** [comment]
<img width="960" alt="Screenshot 2022-03-02 092749" src="https://user-images.githubusercontent.com/80608220/156294109-83e593d3-7ea2-4f5a-b5f3-5df8dd88b4ec.png">
  
sir can i increass it limit 20 to 30

**Comment by AryanRajput78** [comment]
Yes if you add the products to the items list you can scrap as many products as you like. Also, there have been some updates to the Product scraper file so check them as well.
